front/src/components/Profile/Profile.tsx

```diff
@@ -0,0 +1,622 @@
+import {
+  Alert,
+  Box,
+  Button,
+  Card,
+  CardContent,
+  Divider,
+  Grid,
+  IconButton,
+  InputAdornment,
+  TextField,
+  Typography,
+  styled,
+  useTheme,
+} from "@mui/material";
+import React, { useEffect, useState } from "react";
+import { UserApi } from "@/services/api/UserApi";
+import SaveIcon from "@mui/icons-material/Save";
+import { Visibility, VisibilityOff } from "@mui/icons-material";
+import { Session } from "next-auth";
+import { getSession } from "next-auth/react";
+
+interface Errors {
+  Email?: string;
+  Mobile?: string;
+  user_name?: string;
+  display_name?: string;
+  role?: string;
+  newPassword?: string;
+  confirmPassword?: string;
+  currentPassword?: string;
+}
+
+interface RowData {
+  id: number;
+  email: string;
+  mobile: string;
+  user_name: string;
+  display_name: string;
+  role: string;
+  password?: string;
+}
+
+const InitialRowData: RowData = {
+  id: 0,
+  email: "",
+  mobile: "",
+  user_name: "",
+  display_name: "",
+  role: "",
+  password: "",
+};
+
+interface AlertState {
+  severity: "success" | "info" | "warning" | "error";
+  message: string;
+}
+
+interface AddProfileModalProps {
+  onEdit: (editedData: RowData) => void;
+}
+
+const bull = (
+  <Box
+    component="span"
+    sx={{ display: "inline-block", mx: "2px", transform: "scale(0.8)" }}
+  ></Box>
+);
+
+export default function ProfilePage() {
+  const theme = useTheme();
+  const [userDetails, setUserDetails] = useState<{
+    id?: number;
+    email: string;
+    mobile: string;
+    user_name: string;
+    display_name: string;
+    role: string;
+    password: string;
+  }>({
+    email: "",
+    mobile: "",
+    user_name: "",
+    display_name: "",
+    role: "",
+    password: "",
+  });
+
+  const [errors, setErrors] = useState<Errors>({});
+  const [user_name, setUserName] = useState(userDetails.user_name);
```

Comment:
Remove the unused variables(user_name, role, email, mobile, rows and password)

Reply:
removed unused variables